components/Table.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Table({children,className}) {
+  return (
+    <div className={className}>
+        {children}
+    </div>
+  )
+}
```

Comment:
I don't think a new component is needed just for this

Reply:
👍